### PR TITLE
avalicearchiveswiki, SocialProfile configuration

### PR DESCRIPTION
**Pull request by Creepario**
I'm not sure if adding new lines is allowed but I wanted to configure the SocialProfile extension a bit.
default settings based of how it seems to be set

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -6654,6 +6654,21 @@
 			'edits' => 10,
 		),
 	),
+	'wgUserProfileDisplay['friends']' => array(
```
[the remark this answers is not in the document]
Comment (Creepario) [reply LocalSettings.php]: both friends and foes are possible

[the remark this answers is not in the document]
Comment (paladox) [reply LocalSettings.php]: I doint think so with the way we have the config set.

Though for consistency @The-Voidwalker comment looks better :)

File: LocalSettings.php [hunk 2]
```diff
@@ -6654,6 +6654,25 @@
 			'edits' => 10,
 		),
 	),
+	'wgUserProfileDisplay' => array(
```
Comment (paladox) [inline LocalSettings.php]: Needs to be indented correctly below.

File: LocalSettings.php [hunk 3]
```diff
@@ -6654,6 +6654,25 @@
...
+        	'default' => array(
+                'friends' => false,
+		'foes' => false,
+      	  	),
```
Comment (paladox) [inline LocalSettings.php]: add missing ),?

File: LocalSettings.php [hunk 4]
```diff
@@ -6654,6 +6654,25 @@
...
+      	  'avalicearchiveswiki' => array(
+      	          'friends' => true,
+     	   	),
+	),
+	'wgUserBoard' => array(
+		'default' => false,
+		'avalicearchiveswiki' => true,
+		),
```
Comment (paladox) [inline LocalSettings.php]: remove ), ?

**Comment by paladox** [comment]
Tests pass! Rolling out, thanks!.

**Pull request merged**